core/pkg/observability/printer.go

```diff
@@ -52,7 +52,7 @@ func (p *Peeker) Peek(_ *http.Request, resp *http.Response) {
 		return
 	}
 
-	if resp.StatusCode >= http.StatusOK {
+	if resp.StatusCode != http.StatusOK {
```

Comment:
Should this also ignore other 200-level codes, which all indicate success?

Reply:
yeah... just pushed a fix

Reply:
oh... ok... i can modify it...